about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/mobile.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:wght@100;200;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Lato:wght@100;300;400;700;900&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Caveat:wght@400;500;600;700&family=Lato:wght@100;300;400;700;900&family=Poppins&display=swap" rel="stylesheet">
+    <title>About</title>
+</head>
+<body>
+    <div class="top-black-navbar">
+        <div class="space"> </div>
+        <ul class="topbar">
+            <li><a href="https://www.facebook.com/shakespear.samu"><img src="img/icons8-facebook-30.png" alt="facebook"></a></li>
+            <li><a href="https://twitter.com/Taku16671414"><img src="img/icons8-twitter-30.png" alt="twitter"></a></li>
+            <li><a href="https://www.instagram.com/taqu95/?next=%2F"><img src="img/icons8-instagram-30.png" alt="instagram"></a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">logout</a></li>
+        </ul>
+        </div>
```

Comment:
- [ ] KIndly make sure that the top navigation bar is wrapped into the `nav` tag and after that move that `nav` container into the `header` tag. :wink: